evm/v0.5/test/support/LinkToken.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
Is this supposed to be checked in?

Reply:
Yeah it is, this is the original `linkToken.ts` converted to json instead